awscli/examples/cloudwatch/put-metric-stream.rst

```diff
@@ -0,0 +1,17 @@
+**To create a metric stream**
+
+The following ``put-metric-stream`` example creates a metric stream named ``QuickFull-GuaFb`` in the specified account. ::
```

Comment:
Maybe an extra space between the `.` and the `::`.

Reply:
Confirmed that only 1 space here